### PR TITLE
Add license docs

### PR DESCRIPTION
**Pull request by TerryHowe**

### CODE REVIEW

File: docs/content/en/docs/tasks/cluster/cluster-license.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: "License cluster"
+linkTitle: "License cluster"
+weight: 20
+date: 2021-12-10
+description: >
+  How to license your cluster.
+---
+
+If you are are licensing an existing cluster, apply the following secret to your cluster:
```
Comment (chrisnegus) [inline docs/content/en/docs/tasks/cluster/cluster-license.md:10]: Is "my-license-here" replaced by a license number or similar? If so, maybe add "..., apply the following secret to your cluster (replacing `my-license-here` with your license number):"

File: docs/content/en/docs/getting-started/production-environment/_index.md [hunk 2]
```diff
@@ -54,6 +54,14 @@ Make sure you use single quotes around the values so that your shell does not in
    export EKSA_VSPHERE_PASSWORD='t0p$ecret'
    ```
 
+1. Set License Environment Variable
+
+   If you are creating a licensed cluster, set and export the license variable:
```
Comment (chrisnegus) [inline docs/content/en/docs/getting-started/production-environment/_index.md:59]:
```suggestion
   If you are creating a licensed cluster, set and export the license variable (see [License cluster]({{< relref "/docs/tasks/cluster/cluster-license" >}}) for an example of a cluster license secret):
```
I'm assuming they need to apply the cluster license secret first. If so, perhaps add a link like above.

**Review by chrisnegus — COMMENTED** [review]
Looks good. Just a couple of small suggestions.

File: docs/content/en/docs/tasks/cluster/cluster-license.md (hunk 1 above)
Comment (TerryHowe) [reply docs/content/en/docs/tasks/cluster/cluster-license.md]: Done.

**Pull request merged**